helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -433,6 +432,92 @@ public void removeDeploymentGroup(final String name) throws DeploymentGroupDoesN
     }
   }
 
+  public List<RollingOperation> getRollingOperations(final String groupName)
+    throws DeploymentGroupDoesNotExistException {
+    final ZooKeeperClient client = provider.get("getRollingOperations");
+    final List<RollingOperation> ops = new ArrayList<RollingOperation>();
```

Comment:
`ArrayList<RollingOperation>` can be `ArrayList<>` since java7+ has type inference for generic instance creation.